api/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         hikariCPVersion = '3.3.1'
         flywayVersion = '5.2.4'
         postgresJdbcVersion = '42.2.5'
-        jacksonDatabindVersion = '2.9.9.1'
+        jacksonDatabindVersion = '2.9.9.3'
```

Comment:
2.10.0.pr3 caused an error for some reason, didn't want to spend too much time investigating why as it's unrelated to the story.